backend/src/main/java/codezap/template/service/TemplateService.java

```diff
@@ -76,6 +76,15 @@ public Template update(
         return template;
     }
 
+    private void checkCategoryChanged(Category category, Template template) {
```

Comment:
```suggestion
    private void updateCategoryCount(Category newCategory, Template template) {
```

메서드명과 달리 단순 체크만 하지는 않고 내부 로직이 있어서 변경 제안해요!

84, 85라인 변수명도 조금 헷갈릴 수 있을 것 같아서 더 명확하게 제안해봅니당~!

Reply:
변수명에 `new`가 들어가는게 묘하게 불편해서 `changed`로 변경하였습니다!